RELEASING.md

```diff
@@ -0,0 +1,80 @@
+# Cutting New Releases
+
+We chose a manual process which is pretty simple and flexible because of our conventional commit messages.
+
+If you have NPM privileges, you have the ability to cut new releases of the packages herein.
+
+## Requirements
+
+You'll need:
+
+- `GH_TOKEN` : a github user token as an environment variable
+- `git` command line installed, obviously!
+- to run `yarn adduser` first to ensure you're authenticated for publishing
+- your npm 2FA should be enabled, and you should have your second factor device handy for publish
+- your remote should be named `origin` for github, and should be the ssh url
+- (coming soon) GPG key uploaded to account for signing
+
+Note: Ideally we can avoid publishing from any branch but `master`, but we can always `--allow-branch mybranch` in case of an emergency for pre-releases. _Whenever you can, always publish from `master`_.
```

Comment:
add note about publishing, for example, patch versions off of previous minor/major versions is possible using tags using any of the release?